### ISSUE TITLE
[BUG]: Fan control doesn't seem to work on Legion 5 Pro 2023

### ISSUE THREAD

**Issue by noteteszt** [BartoszCichecki/LenovoLegionToolkit#699]
### Version

2.12.0

### OS

Windows 10 22H2 19045.2846

### Device

Legion 5 Pro 2023

### BIOS version

KWCN32WW

### Which features of the application did you test?

It seems everything's working except fan control.

### Which features of the application do not work as expected?

No effect of fan control.

**Comment by BartoszCichecki** [comment]
Let's do a quick test: move all sliders to 100% and hit Save. Is there any difference?

**Comment by noteteszt** [comment]
Sure, I tried the Maximum speed fan switch but nothing. I tried now move up to max the sliders, same results, fans are also about only 2000 rpm. :(

**Comment by BartoszCichecki** [comment]
So just to be 100% clear:

1. You moved all sliders to 100%
2. Clicked Save
3. Nothing happened
4. Checked `Maximum fan speed`
5. Clicked Save
6. Nothing happened

Is that correct?

**Comment by noteteszt** [comment]
Now it's working, I disabled the vantage. I will test in a game. But with no load the CPU temp is about 40C and the fans are working about 2000RPM so the 0RPM part is not working properly.

In the legion 5 pro 2021 I used with 1000 rpm with LFC under 50C without errors for 1,5 year. I hope you can find a solution. Thank you in advance!

**Comment by BartoszCichecki** [comment]
Hmm I was pretty sure I put a warning on top of the page that Vantage needs to be disabled...

**Comment by BartoszCichecki** [comment]
Regarding the 0RPM speeds, it is technically not allowed by Lenovo but confirmed working by many, in some scenarios. With very little load and low temps fans should stop, but if they don't I can't do anything about it.

There is also no way to customize speeds (like settings 1000RPM).

**Comment by noteteszt** [comment]
Hi Bartosz,

Everything fine, except that the CPU temp is 36C but the fans are working continuously. I can't off them to 0 RPM. Do you have any idea?

Thank you very much in advance!

**Comment by BartoszCichecki** [comment]
As I wrote before:

> Regarding the 0RPM speeds, it is technically not allowed by Lenovo but confirmed working by many, in some scenarios. With very little load and low temps fans should stop, but if they don't I can't do anything about it.

**Comment by smazhevskiy** [comment]
> Hi Bartosz,
> 
> Everything fine, except that the CPU temp is 36C but the fans are working continuously. I can't off them to 0 RPM. Do you have any idea?
> 
> Thank you very much in advance!

hi. I recently bought the same laptop with the same problem as yours. Do you have a solution to this problem?

**Comment by noteteszt** [comment]
> > Hi Bartosz,
> > Everything fine, except that the CPU temp is 36C but the fans are working continuously. I can't off them to 0 RPM. Do you have any idea?
> > Thank you very much in advance!
> 
> hi. I recently bought the same laptop with the same problem as yours. Do you have a solution to this problem?

Hi Smaz,

No, I haven't found any solution for the idle noise, I'm use to it. But ingame I just use about 60W power states in custom mode and the fans working on 2900rpm. It's acceptable.

**Comment by smazhevskiy** [comment]
![3scr](https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/78866710/92616e5e-1616-44f2-98f5-46849921f74a)
![2scr](https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/78866710/d1be6439-65b4-405c-8726-ad66afc03c6f)
![1scr](https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/78866710/a66f3322-64f4-4e33-8fd0-2ff95ebc51e8)


I have a solution that might help a little. I downloaded the Lenovo Legion Toolkit program and the RPM is almost always 1700 and the temperature does not rise. I limited the maximum processor temperature to 85 degrees Celsius (useful in games to reset frequencies), BUT I don't know why, but the settings in the program don't always work and the laptop doesn't respond to them, and then it can work normally. I don't know what this is about. I guess as with Windows updates. Because I reinstalled Windows 3 times in a week, it was 11, now I decided to stay on Windows 10. But I still don't understand why sometimes the settings stop working. I had something good, and then I changed the backlight of the keyboard and again the rpm rose to 2100. At the moment, the silent mode works best (and sometimes it does not respond to it at all and it is better to switch to manual adjustment mode). I suppose this is a terrible flaw of the Lenovo developers who do not solve the problem in any way. There are a lot of complaints about the operating mode of the fans. I hope this will help you and you will be able to use your laptop comfortably. I'm sorry for the English. I answer with the help of a translator.
